components/html-block/test/html-block.test.js

```diff
@@ -52,7 +52,7 @@ class TestAsyncRenderer {
 
 class TestNoInlineRenderer {
 	async render(elem, options) {
-		if (options.noDeferredRendering) return;
+		if (options.noDeferredRendering) return elem;
```

Comment:
This wasn't failing any tests because it was the last renderer in the chain, but not returning the element in this case definitely breaks things now that I've added the extra renderer. 😅